packages/client/src/proxy.js

```diff
@@ -168,7 +168,7 @@ export class ProxyHttpsAgent extends https.Agent {
 
     let handleError = err => {
       socket.destroy(err);
-      logger('client:proxy').error(`Proxying request failed: ${err}`);
+      logger('client:proxy').error(`Proxying request ${href(options)} failed: ${err}`);
```

Comment:
don't think this is needed as line 153 logs the proxying request.
Added because it would be better to understand error logs.